src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.model;
+
+public class User {
+    private String name;
+    private String password;
+    private int age;
+    private int id;
+
+    public User(String name, int age) {
```

Comment:
it will be nice to think about some logic of getting id if you want to use it
id should be generated at the time of object creation